integration/flink/flink1/src/main/java/io/openlineage/flink/OpenLineageFlinkJobListener.java

```diff
@@ -88,6 +89,7 @@ static class OpenLineageFlinkJobListenerInternalBuilder
     @Override
     public OpenLineageFlinkJobListener build() {
       Validate.notNull(super.executionEnvironment, "StreamExecutionEnvironment has to be provided");
+      Validate.isTrue(!ClassUtils.hasFlink2Classes(), "Listener can be used with Flink 1.x. Flink 2.x has been detected");
```

Comment:
If we want this to be actionable for the user, can we change that message, for example to:

```
OpenLineageFlinkJobListener detected Flink 2 classes. This integration method is not supposed to work with Flink 2 - please check docs https://openlineage.io/docs/next/integrations/flink/flink2 
```